sdk/textanalytics/Azure.AI.TextAnalytics/perf/Azure.AI.TextAnalytics.Perf/Infrastructure/TextAnalyticsTest.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Test.Perf;
+
+namespace Azure.AI.TextAnalytics.Perf
+{
+    public abstract class TextAnalyticsTest<TOptions> : PerfTest<TOptions> where TOptions : PerfOptions
+    {
+        public TextAnalyticsTest(TOptions options) : base(options)
+        {
+            if (options.Parallel > 1)
+            {
+                throw new InvalidOperationException("Do not stress the service until we can reduce testing costs. Set '--parallel 1'.");
```

Comment:
question: why does the perf test not allow parallel testing? In python, we're running the tests with 50 parallel threads

Reply:
I wonder if this just came as a copy of FR. If other languages are doing parallel threads, then we should do the same.

Reply:
I think people did not want to hit the rate limit quickly. I think you will need to enable it though and do some fine tuning to figure out the right number of parallel threads needed.